src/reducers/drawingState.js

```diff
@@ -15,6 +15,9 @@ const initialState = {
     selectionBoxes: {},
     lastSavedShapes: {},
     editInProgress: false,
+    matrix: [1, 0, 0, 1, 0, 0],
```

Comment:
Maybe a more descriptive name for this? scalingMatrix or canvasTransformationMatrix? Whatever you think would describe the function of the matrix

Reply:
Yes, good call! I'll change it to canvasTransformationMatrix